.circleci/Dockerfile

```diff
@@ -14,6 +14,10 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
 
 RUN apt-get install -y nodejs
 
+RUN mv /usr/bin/docker-compose /usr/bin/docker-compose-v1
+ARG DOCKER_COMPOSE_VERSION=v2.2.2
```

Comment:
Just checking: we don't have a hard dependency on v2 now, do we?

Reply:
Not that I am aware of -- this is to allow us to test primarily against v2. If the fallback CLI loader is used and `which docker-compose` still returns a v1 composer, we'd still use that.

Reply:
Do we think it would be useful to still run some CI against v1?